training/COMPARISON.md

```diff
@@ -0,0 +1,462 @@
+# Arkouda vs NumPy/Pandas
+This document compares and contrasts Arkouda to the similar well known libraries, `NumPy` and `Pandas`. As you go through this document, we encourage you to try the code for yourself. Especially, in the final section.
```

Comment:
Actually looking back at this, im not sure the `;` is the right tool here. iirc, it's supposed to connect independent clauses right? and `especially the final section` cant stand alone. I actually think the dash might be the grammatically correct choice. But it prob doesnt matter all that much since the meaning should be pretty obvious no matter the punctuation we go with

Reply:
done... grammatically, it should be a single `-` so I did that.

Reply:
I'm not sure I agree with the single `-`. I'm prob wrong cause my grammars not super strong but I think we're looking for an em dash here https://www.grammarly.com/blog/dash/

Reply:
based on that source, I agree. Honestly, I asked my wife and she apparently lied to me. I'll update now